makefile

```diff
@@ -113,7 +113,7 @@ include .env
 download:
 	rm -rf $(BAG3D_TEST_DATA)
 	mkdir -p $(BAG3D_TEST_DATA)
-	cd $(BAG3D_TEST_DATA) ; curl -O https://data.3dbag.nl/testdata/pipeline/test_data_v13.zip ; unzip -q test_data_v13.zip ; rm test_data_v13.zip
+	cd $(BAG3D_TEST_DATA) ; curl -O https://data.3dbag.nl/testdata/pipeline/test_data_v14-wip-2.zip ; unzip -q test_data_v14-wip-2.zip ; rm test_data_v14-wip-2.zip
```

Comment:
Is the test data file still work in progress? Why do you call it `-wip-2` instead of just `14` or `15`?